apps/versions/urls.py

```diff
@@ -6,8 +6,10 @@
 
 urlpatterns = patterns(
     '',
-    url('^$', views.version_list, name='addons.versions'),
-    url('^format:rss$', VersionsRss(), name='addons.versions.rss'),
+    url('^(?:status:(?P<status>[^/]+))?$',
```

Comment:
Actually, no - removing `status:` isn't an option, it would break URLs like https://addons.mozilla.org/en-US/firefox/addon/adblock-plus/versions/2.6.10 (`2.6.10` would be considered the status here).